client/asset/dcr/dcr.go

```diff
@@ -6126,7 +6157,7 @@ func (dcr *ExchangeWallet) idUnknownTx(ctx context.Context, ltxr *ListTransactio
 		}, nil
 	}
 
-	if isMix, mixDenom := isMixTx(tx.MsgTx); isMix {
+	if isMix, mixDenom := isMixTx(tx.MsgTx); isMix || isMixedSplitTx(tx.MsgTx) {
```

Comment:
Oh, I guess this isn't a great place since we won't have the mixDenom...

Reply:
Moved to its own section https://github.com/decred/dcrdex/compare/867451f6f5eb71e76ed7537b1a73277e276a802f..909e4b5a918c4b1ad7797f7fd5cce6128ea4b6a6